src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -57,6 +57,10 @@ public Http2Connection(HttpConnectionPool pool, SslStream stream)
             _nextStream = 1;
         }
 
+        public bool UseCookies => _pool.Settings._useCookies;
```

Comment:
These don't need to be public. They should be private.

Actually my suggestion would be to just add a (private) property for Settings. I'm not sure there's a ton of value in having individual properties for the various settings.

Reply:
> Actually my suggestion would be to just add a (private) property for Settings. I'm not sure there's a ton of value in having individual properties for the various settings.

In HttpConnectionSettings class, we already have internal `_useCookies` and `_cookieContainer` fields. I agree that we should reduce individual properties for various settings, will remove those new properties.